src/booleans.js

```diff
@@ -1,63 +1,123 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  if (a === true && b === true) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
If we dig a bit deeper into what this test is doing, this can be simplified a great deal. 
While the logic is sound, it could be simplified like so:
```
  if (a && b ) {
    return true;
  } else {
    return false;
  }
```
Since `a` and `b` are either true or false, you can compare them directly without having to assert whether or not they match true or false.
We can go a step further than this with the rest of the function. Since you are returning either true or false, we can just return the value of the comparison like this:
```
  return (a && b) ? true : false;
```
Furthermore, we can simplify this yet again!
```
  return a && b;
```